src/odemis/driver/npmc.py

```diff
@@ -752,16 +725,21 @@ def _parseMessage(self, msg):
         msg = msg.strip()  # remove leading and trailing whitespace
         msg = "".join(msg.split())  # remove all space characters
         
-        if msg == "ID?":
-            self._sendAnswer("12345")
-
-        elif msg == "VE?":
+        if msg == "VE?":
             self._sendAnswer("1.0")
             
         elif msg == "SM":
             # save memory to non-volatile RAM
             pass
 
+        elif msg == "TE?":
+            # Query error code. Return no error code
+            self._sendAnswer("0")
+
+        # Query the axis ID number
+        elif re.match('\dID\?', msg):
+            self._sendAnswer("12345")
```

Comment:
Can you make something more looking like the actual hardware answer for VE and ID?